File: DropZone/DropZone.Core/Models/Jump/JumpModel.cs
using System;

namespace DropZone.Core.Models
{
    public class JumpModel
    {
        public long Id { get; set; }

        public DateTime ModifiedAt { get; set; }

        public DateTime Date { get; set; }

        public string Task { get; set; }

        public double Height { get; set; }

        public double FallTime { get; set; }


        public long JumpBookId { get; set; }

        public long DropZoneId { get; set; }

        public long ParachuteSystemId { get; set; }

        public long AircraftId { get; set; }


        public JumpBookModel JumpBook { get; set; }

        public DropZoneModel DropZone { get; set; }

        public ParachuteSystemModel ParachuteSystem { get; set; }

        public AircraftModel Aircraft { get; set; }
    }
}

File: DropZone/DropZone.Web/src/models/jumps/jump.model.ts
import { JumpBookModel } from "./jumpBook.model";
import { DropZoneModel } from "../dropzones/dropzone.model";
import { ParachuteSystemModel } from "../parachuteSystems/parachuteSystem.model";
import { AircraftModel } from "../aircrafts/aircraft.model";

export class JumpModel {
	public Id: number;
	public Task: string;
	public Height: number;
	public FallTime: number;
	public Date: Date;
	public ModifiedAt: Date;

	public JumpBookId: number;
	public DropZoneId: number;
	public ParachuteSystemId: number;
	public AircraftId: number;
};